src/App.js

```diff
@@ -1,12 +1,16 @@
 import logo from "./logo.svg";
 import "./App.css";
 
-const PassingProsComponent = ({ children, ...props }) => (
-  <div>
-    {children}
-    {...props}
-  </div>
-);
+const PassingProsComponent = (props) => {
+  const { isDisabled, ...rest } = props;
```

Comment:
**Suggestion:** Consider destructuring props directly in the function parameters. This makes the code cleaner and easier to read.
```suggestion
const PassingProsComponent = ({ isDisabled, ...rest }) => {
```